app/reducers/article.js

```diff
@@ -1,6 +1,6 @@
-import topicJsonMock from '../assets/topic_0_mock.json';
 import { getIntOfLength } from 'utils/math';
 
+// #TODO: create endpoint to return these indices on the back-end
 const ARTICLE_INDEX_ARRAY = [0, 9, 11, 38, 53, 55, 202, 209, 236, 259];
```

Comment:
just a note, there will not be a data structure like `ARTICLE_INDEX_ARRAY` with a list of articles to process on the front-end. There will be an API endpoint for requesting an article to work on.